weedle2/src/lib.rs

```diff
@@ -47,7 +32,7 @@ use self::literal::StringLit;
 use self::mixin::MixinMembers;
 use self::namespace::NamespaceMembers;
 use self::types::{AttributedType, ReturnType};
-pub use nom::{error::Error, error::ErrorKind, Err, IResult};
+use nom::{error::Error, Err, IResult};
```

Comment:
The `Parse` trait is public and uses the `IResult`, `Err` and `Error` are used in the `parse` function, we should continue exporting them.

Reply:
That makes this a breaking change though. I still find it unusual to bump the version in this PR rather than at the point of release though.

Reply:
so, is `3.1.0` a good version for us if I remove the breaking changes? I just follow the #1268, and it said an version bump is needed if `nom` upgrades.

Reply:
The breaking changes cannot be removed as bumping `nom` is itself a breaking change, since its types are part of the public interface. I would recommend not touching the version of uniffi crates at all and leaving that up to the person who does the release.

Reply:
Yeah, doing a breaking change is fine. We basically maintain `weedle2` for use in UniFFI, but we can still be good citizens and do it properly.